docs/getting-started-guides/docker.md

```diff
@@ -1,6 +1,8 @@
 ---
 ---
 
+**Note: [Minikube](../minikube/) is the recommended method of running Kubernetes on your local machine.**
```

Comment:
Clarify that this page is not Minikube.  e.g. `Stop.  This guide has been superseded by Minikube which is the recommended...`